server/publications/channelAutocomplete.coffee

```diff
@@ -13,15 +13,12 @@ Meteor.publish 'channelAutocomplete', (name) ->
 		limit: 5
 
 	cursorHandle = RocketChat.models.Rooms.findByNameContainingAndTypes(name, ['c'], options).observeChanges
-		added: (_id, record) ->
-			pub.added('channel-autocomplete', _id, record)
-
+        added: (_id, record) ->
+			pub.added('channel-autocomplete', _id, record) if _id?
```

Comment:
@marceloschmidt  Apology for that.  My code raise exceptions on undefined _id.
I added this  * if _id? * part to prevent it as I haven't gotten the source of the exception yet.
However after it disappeared after I update the code.